### PR TITLE
Bluewr

### PR DESCRIPTION
**Pull request by wereinse**
## Issues Closed or Referenced

- Closes #issue_number (this will automatically close the issue when the PR closes)
- References #issue_number (this references the issue but does not close with PR)

# Type of PR

- [ ] Documentation changes
- [ x] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## PR Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- x ] My code follows the code style of this project.
- [ ] I ran lint checks locally prior to submission.
- [ ] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Does this introduce a breaking change

- [ ] YES
- [x ] NO

## Validation

- [ ] Unit tests updated and ran successfully
- [ ] Update documentation or issue referenced above

### CODE REVIEW

File: flux.yaml [hunk 1]
```diff
@@ -0,0 +1,173 @@
+apiVersion: v1
```
Comment (bartr) [inline flux.yaml:1]: this file should not be checked in - it is generated during setup

File: gitops/ingress/02-traefik-config.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
+apiVersion: aadpodidentity.k8s.io/v1
```
Comment (bartr) [inline gitops/ingress/02-traefik-config.yaml:1]: this file should not be checked in - it is generated during setup

File: gitops/ngsa/ingress-bluewr.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+apiVersion: networking.k8s.io/v1
```
Comment (bartr) [inline gitops/ngsa/ingress-bluewr.yaml:1]: this file should not be checked in - it is generated during setup

adding this file to the repo will break Traefik for anyone who creates a branch

File: README.md [hunk 4]
```diff
@@ -188,7 +188,8 @@ az keyvault set-policy --secret-permissions list get --object-id $(az ad signed-
 # create AKS
 az deployment group create -g rg-bu0001a0008-$ASB_TEAM_NAME \
   -f cluster-${ASB_TEAM_NAME}.json \
-  -p targetVnetResourceId=${ASB_RESOURCEID_VNET_CLUSTERSPOKE} \
+  -p asbTeamName=${ASB_TEAM_NAME} \
```
Comment (bartr) [inline README.md:191]: alignment is off by one space

Comment (wereinse) [reply README.md:191]: ok

File: flux.yaml (hunk 1 above)
Comment (wereinse) [reply flux.yaml:1]: it's gone now.

File: gitops/ingress/02-traefik-config.yaml (hunk 2 above)
Comment (wereinse) [reply gitops/ingress/02-traefik-config.yaml:1]: it's gone now.

File: gitops/ngsa/ingress-bluewr.yaml (hunk 3 above)
Comment (wereinse) [reply gitops/ngsa/ingress-bluewr.yaml:1]: it's cleaned up now

**Review by bartr — APPROVED** [review]

**Pull request merged**